tests/testthat/test-scatter_plot.R

```diff
@@ -6,32 +6,82 @@ test_data_scatter2 <- data.frame(x = c("blue", "red", "yellow"), y = c(3,2,1))
 
 test_data_scatter3 <- data.frame(x = c(1,1,1), y = c("yes", "no", "yes"))
 
-helper_scatter <- ggplot2::ggplot(test_data_scatter,ggplot2::aes(x=x,y=y)) +
+helper_scatter_empty_type <- ggplot2::ggplot(test_data_scatter,ggplot2::aes(x=x,y=y)) +
   ggplot2::geom_point()+
   ggplot2::labs(x="xlab",y="ylab")+
   ggplot2::ggtitle("Scatterplot Test")+
-  ggplot2::theme(text=ggplot2::element_text(size=10))
-helper_scatter
+  ggplot2::theme(text=ggplot2::element_text(size=10)) + 
+  ggplot2:: geom_smooth(method = NULL, se = FALSE)
```

Comment:
Does `method = NULL` work here/what kind of plot does it look like? Might want to just not include`geom_smooth()` for the regular scatter plot without a model.